src/peft/tuners/waveft/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024-present the HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .config import WaveFTConfig
+from .layer import WaveFTLayer, WaveFTLinear
+from .model import WaveFTModel
+
+
+__all__ = ["WaveFTConfig", "WaveFTLayer", "WaveFTLinear", "WaveFTModel"]
```

Comment:
Here we also need to call `register_peft_method`, see e.g. how it's done here:

https://github.com/huggingface/peft/blob/d6dbbc919576d92c31d00994e41cb503ff132c94/src/peft/tuners/fourierft/__init__.py#L24